tensorflow-proto/tensorflow-proto.cabal

```diff
@@ -4,16 +4,17 @@ synopsis:            TensorFlow protocol buffers.
 description:         Please see README.md
 homepage:            https://github.com/tensorflow/haskell#readme
 license:             Apache
+license-file:        LICENSE
 author:              TensorFlow authors
 maintainer:          tensorflow-haskell@googlegroups.com
 copyright:           Google Inc.
 category:            Machine Learning
 build-type:          Custom
 cabal-version:       >=1.24
-extra-source-files:  ../third_party/tensorflow/tensorflow/core/framework/*.proto
-                   , ../third_party/tensorflow/tensorflow/core/protobuf/config.proto
-                   , ../third_party/tensorflow/tensorflow/core/protobuf/debug.proto
-                   , ../third_party/tensorflow/tensorflow/core/util/event.proto
+extra-source-files:  third_party/tensorflow/tensorflow/core/framework/*.proto
```

Comment:
Yes, for this first release I was planning to include the .proto files in the tarball we upload to Hackage.

Longer-term I'd like to upload the generated `*.hs` files themselves, to avoid the dependency on `protoc`.  But that will require more changes to `Data.ProtoLens.Setup`.